src/status_im/fleet/core.cljs

```diff
@@ -1,17 +1,22 @@
 (ns status-im.fleet.core
   (:require
+    [native-module.core :as native-module]
     [re-frame.core :as re-frame]
     [status-im.multiaccounts.update.core :as multiaccounts.update]
     [status-im.node.core :as node]
-    [status-im2.config :as config]
     [status-im2.constants :as constants]
     [utils.i18n :as i18n]
-    [utils.re-frame :as rf]))
+    [utils.re-frame :as rf]
+    [utils.transforms :as transforms]))
+
+(def default-fleets (transforms/json->js (native-module/fleets)))
+(def default-fleet (.-defaultFleet default-fleets))
```

Comment:
@cammellos, a tradeoff you can make here to remove the cost at startup completely is to compute the fleets in a subscription or just make all three vars functions and call `(fleets/fleets)` in `fleet-settings`.

Perhaps this is the best we should do because most users won't navigate or care about the fleet settings, and it's fine there to compute from scratch, the perf hit should be minimal and not impact other parts of the app.